replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -41,12 +41,28 @@
 
   private static final String SUSPENDED_KEY = "suspended";
 
+  private static final String DELETED_KEY = "deleted";
+
+  private static final String DELETE_DATA_KEY = "deleteData";
+
   /**
-   * 0/No Version - initial version of configs which were saved in the catalog framework. 1 - the
-   * first version of configs to be saved in the replication persistent store.
+   * Field specifying the version of the configuration. Possible versions include:
+   *
+   * <ol>
+   *   <li>0 (No version) - initial version of configs which were saved in the catalog framework
+   *   <li>1 - The first version of configs to be saved in the replication persistent store
+   *       <ul>
+   *         <li>Add <b>suspended</b> field of type boolean
```

Comment:
✏️ Add in the description of both new fields that it defaults to `false` for previous versions.
❗️ Document `deleteData` as well